content/docs/winio/widgets.md

```diff
@@ -0,0 +1,375 @@
+# Widgets
+
+This section covers all provided components in `winio`.
+
+In this section, all methods are expressed as `fn method`, and all "properties" are expressed as `prop property: type`.
+There's no property syntax in Rust, so they are actually `fn property` (or `fn is_property` for `bool` properties) and `fn set_property`.
+A read-only property is expressed as `prop property: const type`.
```

Comment:
Maybe `prop property: type` for immutable properties and `prop property: mut type` for mutable ones? Feels more rust-ish.